pinot-distribution/pinot-assembly.xml

```diff
@@ -123,11 +123,17 @@
     </file>
     <!-- End Include Pinot Input Format Plugins-->
     <!-- Start Include Pinot Minion Tasks Plugins-->
+    <file>
```

Comment:
Switched the comments of minion and metrics plugins

Reply:
Good catch! 👍  Updated.